__tests__/integration/utils/wallet-helper.js

```diff
@@ -27,6 +27,20 @@ export class WalletHelper {
    */
   #started = false;
 
+  /**
+   * Creates a wallet object but does not start it on server
+   * @param {string} walletId
+   * @param {string} [words] 24 words
+   */
+  constructor(walletId, words) {
+    if (!walletId) {
+      throw new Error(`Wallet must have a walletId`);
```

Comment:
Use single quotes

Reply:
As we've discussed on gather, we should use the wallet-service's eslint to keep consistency between projects

Reply:
For now, while we don't implement the linter on #41 , I've changed the code manually to comply with this `single-quote` rule